lib/condor.py

```diff
@@ -266,6 +266,7 @@ def ran_vault_storer_recently(schedd: str, handle: str, outbase: str) -> bool:
     and it was within the last week minus a bit"""
     try:
         sv = os.stat(f"{outbase}/.cvs_{schedd}_{handle}")
+        # print(f"saw cvs file, sv.st_mtime is {sv.st_mtime} check: {sv.st_mtime > time.time() - 604500}")
         # 1 week: 24 * 7 * 60 * 60 = 604800, les 5 minutes (300) -> 604500
         return sv.st_mtime > time.time() - 604500
```

Comment:
Do we think 5 minutes shy of 7 weeks is a good enough buffer, or should we make it six days?

Reply:
Meh,.  Six is fine, and makes for a simpler comment.